contrail-introspect-cli/main.go

```diff
@@ -142,6 +158,24 @@ func DescMpls() DescCollection {
 	}
 }
 
+func routeSummaryDetail(e Element) {
+	table := uitable.New()
+	table.MaxColWidth = 80
+	table.AddRow("Name", "Prefixes", "Paths", "Primary paths", "Secondary paths", "Pending Updates")
+	fields := []string{"name", "prefixes", "paths", "primary_paths",
+	                   "secondary_paths", "pending_updates"}
+	paths, _ := e.node.Search("ShowRouteTableSummary")
```

Comment:
If the collection is correctly build, you then just have to do:
e.node.Search(".")